test/Generator/ContactInfoTest.php

```diff
@@ -44,14 +45,6 @@ public function testSet()
         }
     }
 
-    public function testNull()
```

Comment:
why is this one removed?

Reply:
That is because the following snippet is already tested in `testSet()`:
``` PHP
        foreach (ContactInfo::GETTERS as $modifier) {
            $alice->$modifier();
        }
```